build/set-env.ps1

```diff
@@ -20,10 +20,27 @@ if ($Env:ENABLE_QIRRUNTIME -ne "false" -and $Env:QIR_DROPS -eq $null) {
 }
 
 If ($Env:DROPS_DIR -eq $null) { $Env:DROPS_DIR =  [IO.Path]::GetFullPath((Join-Path $PSScriptRoot "..\drops")) }
+if ($Env:DROP_NATIVE -eq $null) {
+    $Env:DROP_NATIVE = (Join-Path $PSScriptRoot ".." "xplat")
```

Comment:
Hmm... I think this may have been a bug previously that accidentally worked because of how the e2e builds consume the native binaries, and in fixing the value to be correctly populated in this repo it might not coexist well with the e2e logic. Either way, I think the behavior of copying the simulators to this folder are no longer needed for the existing simulator, but might mean that the OpenSim won't show up in the right places in the e2e build. Let's sync up on this one to make sure we understand which pattern the OpenSim should use, and perhaps have a separate PR to remove redundant logic from within this repo.

Reply:
Sounds good, would much appreciate your help. At the moment, if this line is removed, then the builds pass but make a NuGet package that's missing the required DLL; would love to figure out the right way to do things instead.